content/tutorials/get-ether-and-ocean-tokens.md

```diff
@@ -101,3 +90,21 @@ All Squid libraries have methods to request Ocean Tokens. They work by calling t
 - [The squid-java docs](https://www.javadoc.io/doc/com.oceanprotocol/squid/): click "All Classes" then "AccountsManager" then scroll to the bottom of the Class AccountsManager page where you'll find the `requestTokens()` method.
 
 The [Example Code page](/tutorials/example-code/) has links to example Squid code (in all of the languages), including examples of using the above methods.
+
+### Get Mainnet Ocean Tokens
+
+There were several ways to acquire some of the Mainnet Ocean Tokens in the initial circulating supply, including:
+
+- participation in the seed round
+- participation in the pre-sale
+- participation in the token sale
+- participation in the initial exchange offering
+- completion of a [bounty](/concepts/bounties/)
+
+After [the initial exchange offering on Bittrex International](https://blog.oceanprotocol.com/initial-exchange-offering-of-ocean-protocol-on-bittrex-international-a454688f466a), Mainnet Ocean Tokens became available in the Ethereum Mainnet (_not_ the Ocean Mainnet; there was no Ocean Mainnet at the time).
+
+As of 7 May 2019, there was one exchange in the official list of exchanges listing Mainnet Ocean Tokens (in the Ethereum Mainnet): **Bittrex International**.
```

Comment:
by now, coingecko lists some more: https://www.coingecko.com/en/coins/ocean-protocol

We want to add them? If so, we should also link them up and put them in a list and maybe make this paragraph more generic

Reply:
I asked Sarah about that and she said no. We have an official list of exchanges and we must not comment on or link to other ones.

Reply:
They who shall not be named, got it :-)

Reply:
Looks good to me!